app/components/works/form_component.html.erb

```diff
@@ -9,11 +9,11 @@
       html: {class: "needs-validation work-editor", novalidate: true, multipart: true} do |f| %>
   <% if work_form.errors.present? %>
     <div id="error_explanation" class="alert alert-danger">
-      <h4><%= pluralize(work_form.errors.count, "error") %> prohibited this item from being saved:</h4>
+      <h4><%= pluralize(work_form.errors.size, "error") %> prohibited this item from being saved:</h4>
```

Comment:
Needed to make some reform-related tweaks to get the errors to display correctly.